modules.d/95nvmf/parse-nvmf-boot-connections.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+#
+# Supported formats:
+# nvmf.hostnqn=<hostnqn>
+# nvmf.hostid=<hostid>
+# nvmf.discover=<transport>:<traddr>:<host-traddr>:<trsvcid>
+#
+# Examples:
+# nvmf.hostnqn=nqn.2014-08.org.nvmexpress:uuid:37303738-3034-584d-5137-333230423843
+# nvmf.discover=rdma:192.168.1.3::4420
+# nvmf.discover=fc:auto
+#
+# Note: FC does autodiscovery, so typically there is no need to
+# specify any discover parameters for FC.
+#
+
+parse_nvmf_discover() {
+    local OLDIFS="$IFS"
+    local IFS=:
+    local trtrype="none"
+    local traddr="none"
+    local hosttraddr="none"
+    local trsvcid=4420
+
+    set $fcoe
+    IFS="$OLDIFS"
+
+    set $1
```

Comment:
Does that work after resetting IFS?

Reply:
Indeed, it doesn't. Fixed in the next version.